src/components/Slideover/slideover.scss

```diff
@@ -0,0 +1,36 @@
+.sb-slideover {
+  width: 100%;
+  max-width: 62.4rem;
+  height: 100vh;
+  position: fixed;
+  background-color: $white;
+  box-sizing: border-box;
+  padding: 2.5rem;
+  overflow: auto;
+  right: 0;
+  
+  &__close-button {
+    position: fixed;
+    right: 2.5rem;
+    float: right;
+    border: none;
+    background-color: #FFF;
+    cursor: pointer;
+    padding: 0;
+  }
+  
+  .sb-modal-footer {
```

Comment:
The Slideover component should be used along with any element, alright? So, I believe it's not necessary to add this specific rule for `SbModalFooter` and `SbButton` elements. I believe the `Slideover` is just a box on right which encapsulates any element on it. What do you think?

Reply:
Yes, you are totally correct, I will uncouple the css!